torch/onnx/_constants.py

```diff
@@ -4,8 +4,8 @@
 
 ONNX_BASE_OPSET = 9
 ONNX_MIN_OPSET = 7
-ONNX_MAX_OPSET = 18
-ONNX_TORCHSCRIPT_EXPORTER_MAX_OPSET = 17
+ONNX_MAX_OPSET = 19
```

Comment:
This change triggers opset 18 & 19 unittests for `test_pytorch_onnx_onnxruntime.py` in CI, where reduce ops start to fail.

Reply:
Yeah, I think we will need to take the ORT 1.16 bump first, or can include it here too.

Reply:
The failure is due to TS exporter didn't impl opset 18 reduce op updates tho. 

We can either keep this and add version decorator to each failed test marking 17 as supported max version.
Or revert this change and whitelist the fb8 tests such that only those tests run under 19.

Reply:
@BowenBao let's add the version decorator to the failed tests?